### DISCUSSION TITLE
Upgrading standalone rlgl

### DISCUSSION THREAD

**Discussion by sponge** [raysan5/raylib#1882] [Q&A] [answered]
Hi!

I've been using rlgl standalone to do some basic 2D drawing in OpenGL 3.2, and I've been working my way through upgrading my version of the library from 2018, splitting it up into chunks to make sure things are working along the way.

I've been having trouble with one specific change, and I've narrowed it down to this one:
https://github.com/raysan5/raylib/commit/d4ccca81dbb7f8300aa0b7a6b0f195b1814dc9fc#diff-30abe7ae1669d72f0c978b3e2b30985524f8a9b0d90a19c7bee81134c954c213

Looking at the output in RenderDoc, all vertex positions, tex coords, and colors are 0,0,0 in the vertex shader input. This looks like a fairly large diff though, and I don't see anything obvious that I need to change on my side. Is there anything breaking in here I need to adjust with this change in order to fix my vertex positions from being 0?

rlgl has been great otherwise, thanks for a great library!

**Comment by raysan5** [comment]
@sponge sincerely, I don't know... rlgl went through a big redesign for raylib 3.7 but I verified it worked in the standalone mode example...

**Comment by sponge** [comment, marked as answer]
Mentioning the standalone mode example did it, didn't know this existed! I went through and compared it and immediately found that I just needed a `rlLoadExtensions(&SDL_GL_GetProcAddress);` before `rlglInit`.

**Reply by raysan5** [reply]
Nice! Glad it worked! 👍😄
